graphrag/index/api.py

```diff
@@ -24,8 +22,10 @@
 
 async def build_index(
     config: GraphRagConfig,
-    run_id: str,
-    memory_profile: bool,
+    run_id: str = "",
+    is_resume_run: bool = False,
+    is_update_run: bool = False,
```

Comment:
Can both `is_resume_run` and `is_update_run` be True?

Reply:
At the time, yup. I was thinking adding a validation in the cli, but, makes sense to have it here as well